exporter/prometheusremotewriteexporter/helper.go

```diff
@@ -134,6 +166,24 @@ func createLabelSet(labels []*common.StringKeyValue, extras ...string) []prompb.
 		}
 	}
 
+	for _, ra := range resourceAttributes {
```

Comment:
Should this for loop go before the previous? I'm assuming we want user defined labels to override resource attribute labels by default (rather than vice versa).

Reply:
Agreed, they should be the other way around, will update

Reply:
It might be nice to add a test for that too although I'm not sure how necessary that would be.

Reply:
for sure it will be useful to check overriding labels, I will add those as well